examples/targeting/search_for_language_and_carrier_constants.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+# Copyright 2018 Google LLC
```

Comment:
since this is a new file, make the date 2020

Reply:
Fixed.